modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -286,10 +286,19 @@ public String toDefaultValue(Schema p) {
 
     @Override
     public String toModelImport(String name) {
-        // name looks like cat.Cat
-        String moduleName = name.split("\\.")[0];
-        // https://exceptionshub.com/circular-or-cyclic-imports-in-python.html
-        return "from " + modelPackage() + " import "+ moduleName;
+        String modelImport;
+        if (StringUtils.startsWithAny(name, "import", "from", "try:")) {
+            modelImport = name;
+        } else {
+            String[] moduleStruct = name.split("\\.");
+            // https://exceptionshub.com/circular-or-cyclic-imports-in-python.html
+            modelImport = "    from " + modelPackage() + " import " + moduleStruct[0];
+            modelImport = "try:\n" +
+                                modelImport +
+                          "\nexcept ImportError:\n    " +
+                                moduleStruct[0] + " = sys.modules[\n        '" + modelPackage() + "." + moduleStruct[0] + "']";
+        }
```

Comment:
Can we replace this code with robustImport?

Reply:
I did it this way so I changed as little of the original code as possible, but I agree it would be best to use `robustImport` if it was going to be kept.
I an not certain now, but I think I thought at the time I had to modify `robustImport` to make it work and I didn't want to alter any of the code (see response to your previous comment.
Again, your call. Tell me what you want.

Reply:
It looks like robustImport may now be redundant with your update.
Let's remove robustImport and just use toModelImport at all locations if we can.
If we can do that, let's delete robustImport and fixModelImports if we can.
Or if you want to go the minimal changes route, how about not changing toModelImport?
My preference is to do the suggested clean up because it looks like we may be able to remove robustImport and fixModelImports.

Reply:
OK, I will look through the experimental module and clean up the imports properly. I am a bit stretched at the moment, so it wont be for a couple of days, sorry.

Reply:
No worries. We can wait. Of if you want to go the quicker route you could revert the change to this function.